Sources/TuistKit/Services/AuthTokenRefreshService.swift

```diff
@@ -0,0 +1,97 @@
+import FileSystem
+import Foundation
+import Mockable
+import Path
+import ServiceContextModule
+import TuistCore
+import TuistLoader
+import TuistServer
+import TuistSupport
+
+@Mockable
+protocol AuthTokenRefreshServicing: AnyObject {
```

Comment:
```suggestion
protocol AuthTokenRefreshServicing {
```

`AnyObject` conformance is added only if you need to have class semantics when referencing a value when the value's type is of that protocol.